src/docker/composebuilder/index_test.go

```diff
@@ -14,27 +14,6 @@ import (
 
 var _ = Describe("composebuilder", func() {
 	var _ = Describe("GetApplicationDockerConfigs", func() {
-		It("should return the proper docker configs for deployment", func() {
-			appDir, err := ioutil.TempDir("", "")
-			Expect(err).NotTo(HaveOccurred())
-			err = helpers.CheckoutApp(appDir, "rds")
-			Expect(err).NotTo(HaveOccurred())
-			appConfig, err := types.NewAppConfig(appDir)
-			Expect(err).NotTo(HaveOccurred())
-			dockerConfigs, err := composebuilder.GetApplicationDockerConfigs(composebuilder.ApplicationOptions{
-				AppConfig: appConfig,
-				AppDir:    appDir,
-				BuildMode: composebuilder.BuildMode{
-					Type: composebuilder.BuildModeTypeDeploy,
-				},
-			})
-			Expect(err).NotTo(HaveOccurred())
-
-			By("ignoring rds dependencies")
-			delete(dockerConfigs, "my-sql-service")
-			Expect(len(dockerConfigs)).To(Equal(0))
-		})
-
```

Comment:
After pointing `exo deploy` to `run_production.yml`, we were no longer generating a file just for deployment, so this test is irrelevant. But it does bring up one issue we created in switching deploy over to `run_produciton.yml`: rds dependencies will be generated in the docker-compose file (because `exo run --production` will need them presumably), but they shouldn't be pushed to ECR. We might need an additional method on the production dependencies such as `NeedsPush` to call, but I know we're switching to plugin type dependencies, so I'm not sure if that change should be made right now.